src/components/basic/DlPanelContainer/DlPanelContainer.vue

```diff
@@ -435,11 +472,7 @@ export default defineComponent({
                     this.isFullWidth = false
                 }
             } else {
-                if (e.pageX > this.left + this.w) {
-                    this.isFullWidth = true
-                } else {
-                    this.isFullWidth = false
-                }
+                this.isFullWidth = true

```

Comment:
Does it affect other places using the component ?

Reply:
this wont affect other places .